docs/0.4/examples/gameroom/walkthrough/index.md

```diff
@@ -388,3 +422,970 @@ GET /api/proposals
+
+3. The admin service creates a consensus proposal (a `​Proposal​` struct) with the
+   proposal ID, summary, and the list of required verifiers. For more
+   information, see ​section I-2.5.1, step 3​.
+
+##### I-2.5.5. Acme and Bubba Bakery reach consensus
+
+When the admin services have validated the proposal and consensus has reached
+agreement, consensus will notify the admin services to commit the proposal. See ​
+Appendix B​ for more information about consensus.
+
+#### I-2.6. Admin services commit pending circuit proposal
+
+After the consensus notification, both admin services commit the
+`​CircuitProposal​`. Now the new circuit is officially pending, which means that
+the `​CircuitProposal​` is stored in the admin services' state but the circuit is
+not yet available for communication. A pending circuit proposal is also called
+an "open circuit proposal".
+
+In the Gameroom example, the pending circuit ID specifies the application, the
+member nodes, and a version 4 UUID, as in this example:
+
+`gameroom::​acme-node-000​::bubba-node-000::<UUIDv4>`
+
+#### I-2.7. Admin services notify authorization handler of pending circuit proposal
+
+1. After the circuit proposal has been committed, the admin service on each node
+   checks if there are any registered application authorization handlers for the
+   circuit management type in the proposed circuit
+   (​`gameroom::acme-node-000::bubba-node-000::<UUIDv4>`​). See The Prequel,
+   section P.3​, for more information on the registration process.
+
+   An application authorization handler manages the voting strategy for the
+   application and notifies the application of any events received from the
+   admin service of the local Splinter node. This handler registers with an
+   admin service for a specific circuit management type (also described in
+   ​The Prequel, section P.3​).
+
+2. If there are any registered application authorization handlers for the
+   proposed circuit management type, each admin service forwards the request
+   to the local connected Gameroom application authorization handler.
+
+   The notification is sent on a WebSocket connection.
+
+``` json
+{
+"eventType": "ProposalSubmitted",
+"message": {
+  "proposal_type": "Create",
+  "circuit_id": "gameroom::acme-node-000::bubba-node-000::<UUIDv4>",
+  "circuit_hash": "...",
+  "circuit": {
+    "circuit_id":"gameroom::acme-node-000::bubba-node-000::<UUIDv4>",
+    "authorization_type": "Trust",
+    "members": [{
+      "node_id": "acme-node-000",
+      "endpoint": "tls://splinterd-node-acme:8044"
+      },
+    {
+      "node_id": "bubba-node-000",
+      "endpoint": "tls://splinterd-node-bubba:8044"
+      }
+    ],
+    "roster": [{
+      "service_id": "gameroom_acme-node-000",
+      "service_type": "scabbard",
+      "allowed_nodes": [ "acme-node-000" ],
+      "arguments": {
+        "peer_services": [ "gameroom_bubba-node-000" ],
+        "admin_keys": [
+          <acme gameroomd public key>
```

Comment:
Here, too, the maroon text on black is hard to read. If the text color can't be changed, perhaps we should lighten the backgrounds for all code snippets?

Reply:
Updated all of these to remove the json specifier so the formatting looks nicer